static/usage/accordion/multiple/react.md

```diff
@@ -0,0 +1,43 @@
+```tsx
+import React from 'react';
+import { 
+  IonAccordion, 
+  IonAccordionGroup,
+  IonContent,
+  IonItem, 
+  IonLabel
+} from '@ionic/react';
+function Example() {
+  return (
+    <IonContent>
+      <IonAccordionGroup multiple={true} value={['first', 'third']}>
```

Comment:
This is running into this issue: https://github.com/ionic-team/ionic-framework/issues/25041 You may want to set the value via JS instead as a workaround.